sympy/core/basic.py

```diff
@@ -76,6 +76,7 @@ class Basic(with_metaclass(ManagedProperties)):
     is_Number = False
     is_Float = False
     is_Rational = False
+    is_DecimalRational = False
```

Comment:
Instead of adding this we should just use `isinstance`. The `is_Class` stuff in SymPy is not great. I would personally rather it weren't there at all.

Reply:
Fair enough.  I was trying to be consistent with what Rational does, but would you like me to take it out?

Reply:
Yeah, remove `is_DecimalRational` and replace `x.is_DecimalRational` with `isinstance(x, DecimalRational)`. 

The only potential issue with this is you might get import errors from circular import issues, but if that happens, it usually works to move the import inside the function where `DecimalRational` is used.

Reply:
Done.